Python/Location-based-on-IP/get.py

```diff
@@ -0,0 +1,13 @@
+from modulefinder import ModuleFinder
+
+finder = ModuleFinder()
+finder.run_script('get_coordinates.py')
```

Comment:
This is also different issue

Reply:
Okay, I will remove that folder, thanks for the assistance

Reply:
I have removed the files